diff --git a/target/linux/generic/config-4.14 b/target/linux/generic/config-4.14
index aa227ec..098be93 100644
--- a/target/linux/generic/config-4.14
+++ b/target/linux/generic/config-4.14
@@ -2590,7 +2590,7 @@ CONFIG_MII=y
 # CONFIG_MIPS_ALCHEMY is not set
 # CONFIG_MIPS_CDMM is not set
 # CONFIG_MIPS_COBALT is not set
-# CONFIG_MIPS_FPU_EMULATOR is not set
+CONFIG_MIPS_FPU_EMULATOR=y
 # CONFIG_MIPS_GENERIC is not set
 # CONFIG_MIPS_MALTA is not set
 # CONFIG_MIPS_O32_FP64_SUPPORT is not set
