diff --git a/target/linux/generic/config-5.15 b/target/linux/generic/config-5.15
index 214a1e6ecd..7541061621 100644
--- a/target/linux/generic/config-5.15
+++ b/target/linux/generic/config-5.15
@@ -3569,7 +3569,7 @@ CONFIG_MII=y
 # CONFIG_MIPS_COBALT is not set
 # CONFIG_MIPS_CPS is not set
 # CONFIG_MIPS_ELF_APPENDED_DTB is not set
-# CONFIG_MIPS_FPU_EMULATOR is not set
+CONFIG_MIPS_FPU_EMULATOR=y
 # CONFIG_MIPS_FP_SUPPORT is not set
 # CONFIG_MIPS_GENERIC is not set
 # CONFIG_MIPS_GENERIC_KERNEL is not set