diff --git a/target/linux/generic/config-5.10 b/target/linux/generic/config-5.10
index 4529f3888a..d51fc2e302 100644
--- a/target/linux/generic/config-5.10
+++ b/target/linux/generic/config-5.10
@@ -3398,7 +3398,7 @@ CONFIG_MII=y
 # CONFIG_MIPS_ALCHEMY is not set
 # CONFIG_MIPS_CDMM is not set
 # CONFIG_MIPS_COBALT is not set
-# CONFIG_MIPS_FPU_EMULATOR is not set
+CONFIG_MIPS_FPU_EMULATOR=y
 # CONFIG_MIPS_FP_SUPPORT is not set
 # CONFIG_MIPS_GENERIC is not set
 # CONFIG_MIPS_GENERIC_KERNEL is not set