diff --git a/target/linux/generic/config-6.12 b/target/linux/generic/config-6.12
--- a/target/linux/generic/config-6.12
+++ b/target/linux/generic/config-6.12
@@ -4355,10 +4355,10 @@
 # CONFIG_NFT_TPROXY is not set
 # CONFIG_NFT_TUNNEL is not set
 # CONFIG_NFT_XFRM is not set
-# CONFIG_NF_CONNTRACK is not set
+CONFIG_NF_CONNTRACK=y
 # CONFIG_NF_CONNTRACK_AMANDA is not set
 # CONFIG_NF_CONNTRACK_BRIDGE is not set
-# CONFIG_NF_CONNTRACK_EVENTS is not set
+CONFIG_NF_CONNTRACK_EVENTS=y
 # CONFIG_NF_CONNTRACK_FTP is not set
 # CONFIG_NF_CONNTRACK_H323 is not set
 # CONFIG_NF_CONNTRACK_IRC is not set
